requirements/base.pip

```diff
@@ -10,6 +10,8 @@
     # via -r requirements/base.in
 -e git+https://github.com/onaio/oauth2client.git@75dfdee77fb640ae30469145c66440571dfeae5c#egg=oauth2client
     # via -r requirements/base.in
+-e git+https://github.com/onaio/ona-oidc.git#egg=ona-oidc
```

Comment:
Kindly regenerated the `base.pip` & `dev.pip` files

Reply:
done